### PR TITLE
ACS-8607: Create Helm chart for live-ingester - adding liveness probe endpoint

### PR DESCRIPTION
**Pull request by krdabrowski**
https://hyland.atlassian.net/browse/ACS-8607

### CODE REVIEW

**Review by Pawel-608 — APPROVED** [review]
LGTM

File: live-ingester/src/main/resources/application.yml [hunk 1]
```diff
@@ -1,15 +1,19 @@
-application:
-  source-id: alfresco-dummy-source-id-0a63de491876
-  version: "@project.version@"
+server:
+  port: 8080
```
Comment (mpichura) [inline live-ingester/src/main/resources/application.yml:2]: I'd be surprised if this doesn't result in ports clash when running docker compose from command line.
Can we use a different port by default (or add port mapping to DC files)?

Comment (krdabrowski) [reply live-ingester/src/main/resources/application.yml:2]: Live ingester server port is not exposed outside container, so there shouldn't be any issues with port clashes. We expose only debug port

**Review by mpichura — APPROVED** [review]
LGTM

**Pull request merged**